openedx/features/caliper_tracking/third_party/auth/constants.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
The entire `third_party` folder is unnecessary. You can simply add all of this into the `apps.py` file.

Reply:
done